rustual-boy-cli/src/emulator.rs

```diff
@@ -304,6 +329,16 @@ impl Emulator {
                         println!("Watchpoint at 0x{:08x} does not exist", addr);
                     }
                 }
+                #[cfg(not(feature = "cpu-trace"))]
+                Ok(Command::CpuTrace) => {
+                    println!("cputrace requires --features cpu-trace");
```

Comment:
Good call with the alternate version here :)

nit: I think you may be able to match and just use `cfg` within an `if` statement, if you'd prefer that to having different match arms: http://rustbyexample.com/attribute/cfg.html